pynest/nest/lib/hl_api_simulation.py

```diff
@@ -332,11 +333,11 @@ def SetKernelStatus(params):
         Defines all synapses which are plastic for the structural plasticity
         algorithm. Each entry in the dictionary is composed of a synapse model,
         the pre synaptic element and the postsynaptic element
-    structural_plasticity_update_interval : int
+    structural_plasticity_update_interval : int, default: 10000.0
```

Comment:
In my NEST version (2.20.0) this value defaults to 1000 not 10000, can you double check this value?

Reply:
This appears to be correct,  I get `'structural_plasticity_update_interval': 10000.0,`

Reply:
Alright! Then I guess it was changed in NEST 3